caesars-palace/blocks/stats/stats.js

```diff
@@ -1,8 +1,63 @@
 export default function decorate(block) {
-  const statsContainerElement = [...block.children][0];
+  const statsRows = [...block.children];
 
-  if (statsContainerElement) {
-    const columnsInRow = statsContainerElement.children.length;
-    statsContainerElement.style = `grid-template-columns: repeat(${columnsInRow}, 1fr)`;
+  if (statsRows.length === 1) {
+    // row of stats can be short, so adjusting styling depending on columns count
+    statsRows[0].classList.add('short-stats');
+    const columnsInRow = [...statsRows[0].children].length;
+
+    const mediaSmallWidthQueryMatcher = window.matchMedia('only screen and (max-width: 767px)');
+    const mediaSmallWidthChangeHandler = async (event) => {
+      if (event.matches === false) {
+        statsRows[0].style = '';
+      } else if (columnsInRow < 2) {
+        statsRows[0].style = `grid-template-columns: repeat(${columnsInRow}, 1fr)`;
+      }
+    };
+    mediaSmallWidthChangeHandler(mediaSmallWidthQueryMatcher);
+    mediaSmallWidthQueryMatcher.addEventListener('change', (event) => {
+      mediaSmallWidthChangeHandler(event);
+    });
+
+    const mediaWidthQueryMatcher = window.matchMedia('only screen and (min-width: 768px)');
+    const mediaWidthChangeHandler = async (event) => {
+      if (event.matches === false) {
+        statsRows[0].style = '';
+      } else if (columnsInRow < 3) {
+        statsRows[0].style = `grid-template-columns: repeat(${columnsInRow}, 1fr)`;
+      }
+    };
+    mediaWidthChangeHandler(mediaWidthQueryMatcher);
+    mediaWidthQueryMatcher.addEventListener('change', (event) => {
```

Comment:
Could be simplified.

Reply:
Done. I'm partial to that one - I often prefer the arrow notation even for one liners because it makes it easier to extend the function body later.